src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,38 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_LOGIN_LENGTH = 6;
+    private static final int MIN_PASSWORD_LENGTH = 6;
+    private static final int MIN_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RuntimeException("User cannot be null");
+        }
+        if (user.getLogin() == null) {
+            throw new RuntimeException("Login cannot be null");
+        }
+        if (user.getLogin().length() < MIN_LOGIN_LENGTH) {
+            throw new RuntimeException("Login must be at least 6 element");
+        }
+        if (user.getPassword() == null) {
+            throw new RuntimeException("Password cannot be null");
+        }
+        if (user.getPassword().length() < MIN_PASSWORD_LENGTH) {
+            throw new RuntimeException("Password must be at least 6 characters");
```

Comment:
The exception message should be more informative. Consider specifying the exact minimum length required, e.g., 'Password must be at least ' + MIN_PASSWORD_LENGTH + ' characters'.